scripts/collections-sql/hurricane_ida_planetscope_images.sql

```diff
@@ -0,0 +1,46 @@
+INSERT INTO pgstac.collections (content) VALUES('{
+   "id": "blue-tarp-planetscope",
+   "type": "Collection",
+   "links":[
+   ],
+   "title":"Hurricane Ida - Blue Tarps PlanetScope Image",
+   "extent":{
+      "spatial":{
+         "bbox":[
+            [
+               -90.300691019583,
+               29.791754950316868,
+               -89.86300184384689,
+               30.099979027371006
+            ]
+         ]
+      },
+      "temporal":{
+         "interval":[
+            [
+               "2021-08-23T00:00:00Z",
+               "2022-02-12T00:00:00Z"
```

Comment:
Same question: Should this extend to February of this year?

Reply:
same.